system-addon/test/functional/mochitest/browser.ini

```diff
@@ -1,9 +1,5 @@
 [DEFAULT]
-# XXX This defaults to forcing activity-stream tests to be skipped in m-c,
-# since, as of this writing, mozilla-central itself is still turned off.
-# The tests can be run locally using 'npm run mochitest' which does various
-# overrides.
-skip-if=!activity_stream
+skip-if=!nightly_build
```

Comment:
It's to match the logic of browser/extensions `moz.build` where activity-stream isn't even built not on nightly, so running tests would definitely fail. I filed #2893 to make sure we have the right settings (most likely build and test every channel but pref on by default only in nightly [or potentially early_beta_or_earlier]).